astropy/modeling/tests/test_parameters.py

```diff
@@ -1235,7 +1235,7 @@ def evaluate(self, *args):
             return
 
     # a broadcasts with both b and c, but b does not broadcast with c
-    MESSAGE = r"Mismatch is between arg \d with shape .* and arg \d with" " shape .*"
+    MESSAGE = r"Mismatch is between arg \d with shape .* and arg \d with shape .*"
```

Comment:
Apparently Ruff formatter wasn't able to find this, but [ISC001 (single-line-implicit-string-concatenation)](https://docs.astral.sh/ruff/rules/single-line-implicit-string-concatenation/#single-line-implicit-string-concatenation-isc001) did.